.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk: oraclejdk8
+
+script:
+  - mvn test --batch-mode
+  - mvn checkstyle:check --batch-mode --fail-never
```

Comment:
What's `--fail-never` doing? Shouldn't Travis-CI fail the build ( [failOnViolation](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#failOnViolation) on style regression?